public/dashboard.html

```
@@ -5,62 +5,229 @@
-      <a href="/teach.html" class="hover:text-brand" id="host-link">Host</a>
+    <div class="nav-links">
+      <a href="/" class="nav-link">Activities</a>
+      <a href="/dashboard.html" class="nav-link active">Dashboard</a>
+      <a href="/teach.html" class="nav-link" id="host-link">Host</a>
     </div>
-    <div class="flex items-center gap-3">
-      <span class="text-slate-600 text-sm hidden sm:block" id="nav-user"></span>
-      <button onclick="logout()" class="text-slate-400 text-sm hover:text-red-500 transition">Logout</button>
+    <div id="nav-auth" style="display:flex;align-items:center;gap:10px;position:relative;">
+      <span class="nav-user" style="font-size:.82rem;font-weight:600;color:rgba(255,255,255,.8);text-decoration:none;padding:6px 13px;border-radius:8px;transition:color .15s;" id="nav-user"></span>
+      <button onclick="logout()" style="font-size:.82rem;font-weight:500;color:rgba(255,255,255,.7);text-decoration:none;padding:6px 13px;border-radius:8px;transition:color .15s;background:transparent;border:none;cursor:pointer;" onmouseover="this.style.color='#fff'" onmouseout="this.style.color='rgba(255,255,255,.7)'">Logout</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)